client/config.go

```diff
@@ -193,3 +198,9 @@ func (c Configuration) NewAdminClient(
 
 	return NewAdminClient(opts)
 }
+
+// HashingConfiguration is the configuration for hashing
+type HashingConfiguration struct {
```

Comment:
I would place the struct below `Configuration` and above the parsing methods, just for readability of the file.